Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM quay.io/mozmar/base:latest
+WORKDIR /app
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PIP_DISABLE_PIP_VERSION_CHECK=1
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y --no-install-recommends python2.7 libpython2.7 python-dev \
+    python-pip gettext build-essential \
+    libxml2-dev libxslt1-dev zlib1g-dev git\
+    libjpeg-dev libffi-dev libssl-dev libxslt1.1\
+    libmysqlclient-dev software-properties-common apt-transport-https
+
+# Install mariadb-client 5.5
+RUN apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 0xcbcb082a1bb943db
+RUn add-apt-repository 'deb [arch=amd64,i386] http://sfo1.mirrors.digitalocean.com/mariadb/repo/5.5/debian wheezy main'
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends  mariadb-client-5.5 libmariadbclient-dev
+
+# Install nodejs older version 0.10.x
+RUN curl -sL https://deb.nodesource.com/setup_0.10 | bash -
+RUN apt-get install -y nodejs=0.10.48-1nodesource1~xenial1
+
+# https://github.com/pypa/setuptools/issues/544
+RUN pip install --upgrade setuptools
+# Copy the scripts folder because need to run peep from there
+COPY ./scripts /app/scripts
+COPY ./peep.sh /app/peep.sh
+COPY ./requirements /app/requirements
+COPY ./package.json /app/package.json
+COPY ./bower.json /app/bower.json
+
+RUN pip install pip==7.0
+RUN ./peep.sh install -r requirements/default.txt --no-binary :all:
+
+# Add a non root user and use it further
+RUN adduser -u 1001 --gecos 'kitsune' --no-create-home kitsune
+# Give permission to the created user to write in directory
+RUN chown -R kitsune /app
+# npm and bower tries to write in /home directory. So this permission is needed
```

Comment:
Hmm since there's no `home` for kitsune user what are you trying to achieve here?